Src/Sentry.Xamarin.Forms/Properties/AssemblyInfo.cs

```diff
@@ -1,5 +1,6 @@
 using System;
 using System.Runtime.CompilerServices;
-[assembly: InternalsVisibleTo("Sentry.Xamarin.Forms.Tests.UWP")]
+[assembly: InternalsVisibleTo("Sentry.Xamarin.Forms.UWP.Tests")]
+[assembly: InternalsVisibleTo("Sentry.Xamarin.Forms.Testing")]
 
 [assembly: CLSCompliant(true)]
```

Comment:
![image](https://user-images.githubusercontent.com/1633368/109724851-362dd080-7b7e-11eb-82f0-d2ddcdded1e4.png)

The missing line doesn't bother you? 😆 

The reason it's a good thing is that it avoids the `\n` diff on the line above when you add a new line to the file.

```suggestion
[assembly: CLSCompliant(true)]

```

`.editorconfig` rule helps with that as editors will always add a new line.

Reply:
It doesn't bother me but I'll keep that in mind on the next PRs